dist/ktx2Decoder/package.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I don't know, I have just copied what has been done for the NME.

Note that the case for all the other packages: gui, inspector, loaders, ... can be found in dist/ and dist/preview release

Reply:
NME has been shipped, so it will be in both `dist` and `dist/preview release`. Since this hasn't shipped in a release yet, it should only be in `dist/preview release`.

Reply:
Done.